### PR TITLE
[scalability][presubmit] Add optional Python specific test

### PR DESCRIPTION
**Pull request by jprzychodzen**
/cc @wojtek-t 

Ref. kubernetes/perf-tests#1470

Marking this as an optional test to verify

### CODE REVIEW

File: config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml [hunk 1]
```diff
@@ -603,6 +603,31 @@ presubmits:
             cpu: 1
             memory: "2Gi"
 
+  - name: pull-perf-tests-verify-all-python
+    decorate: true
+    always_run: true
+    optional: true
```
Comment (wojtek-t) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml:609]: Please add a TODO to remove this once we prove it's stable.

File: config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml [hunk 2]
```diff
@@ -603,6 +603,31 @@ presubmits:
...
+    skip_branches:
+      - release-1.1[2-8] # Presubmit has been implemented for 1.19+ releases
```
Comment (wojtek-t) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml:611]: We don't have this script for 1.19 too.

File: config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml (hunk 1 above)
Comment (jprzychodzen) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml:611]: Done.

File: config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml (hunk 2 above)
Comment (jprzychodzen) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml]: Good catch! Thanks

**Pull request merged**